employee_manual/travelling.md

```diff
@@ -0,0 +1,29 @@
+# Travelling
+
+## Commuting to your main office
+Commuting does not count as work time.
+
+However, if your commute distance is long enough you might be elligable for "Deduction for transport to work" (Befordringsfradrag). More information is available at [skat.dk](https://skat.dk) (use the search feature).
+
+## Travelling between offices
+You are NOT expected to work overtime to cover time spent travelling, and Abtion will cover your expenses (get in touch with your PM for further details).
+
+We do however ask you to make your time as productive as possible:
+- Travel by train as far as possible
```

Comment:
Not sure about this one, but maybe word it like "as much as possible"? To me "as far as possible" sounds like it's related to distance :)

Reply:
I took that directly from the other document, I was also not entirely happy about it.